src/Invio.Extensions.Reflection/TypeExtensions.cs

```diff
@@ -43,6 +52,113 @@ public static class TypeExtensions {
 
             return name;
         }
+
+        /// <summary>
+        ///   Determine whether or not the <see cref="Type" /> of interest,
+        ///   <paramref name="type" />, has the <paramref name="parentType" />
+        ///   somewhere in its hierarchy.
+        /// </summary>
+        /// <param name="type">
+        ///   The <see cref="Type" /> instance whose hierarchy will be checked to see
+        ///   if it is the same type as <paramref name="parentType" />, or if it
+        ///   a derivative of <paramref name="parentType" />.
+        /// </param>
+        /// <param name="parentType">
+        ///   The <see cref="Type" /> instance that may or may not be somewhere
+        ///   in the hierarchy of <paramref name="type" />.
+        /// </param>
+        /// <exception cref="ArgumentNullException">
+        ///   Thrown when <paramref name="type" /> or <paramref name="parentType" /> is null.
+        /// </exception>
+        /// <exception cref="ArgumentException">
+        ///   Thrown when <paramref name="type" /> is an open generic <see cref="Type" />.
```

Comment:
Is this actually the case? I don't see where this would be thrown, and it looks like there are tests to the contrary on line 122/123 of the test class.

Reply:
It is not. It was for a previous version of the method, but I altered it without updating the comments.

h/t